docs/source/subcommands.md

```diff
@@ -6,27 +6,27 @@
 $ latch init PACKAGE_NAME
 ```
 
-This command can be used to generate an example workflow ([this workflow](https://genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2009-10-3-r25.pdf), to be specific) for reference purposes. It has all of the [required elements](basics/what_is_a_workflow.md) for a workflow, and can also be used as boilerplate - simply replace the logic with your logic, adding more tasks as necessary.
+This command is used to initialize a new workflow directory, using one of four templates. It will create a directory with the name `PACKAGE_NAME` and populate it with the necessary files to get started.
 
-The parameter `PACKAGE_NAME` is the name of the directory that `latch init` will create and populate with example files. The resulting directory structure is as below:
+The parameter `PACKAGE_NAME` is the name of the directory that `latch init` will create and populate with example files.
 
-```text
-PACKAGE_NAME
-├── Dockerfile
-├── data
-│   ├── wuhan.1.bt2
-│   ├── wuhan.2.bt2
-│   ├── wuhan.3.bt2
-│   ├── wuhan.4.bt2
-│   ├── wuhan.fasta
-│   ├── wuhan.rev.1.bt2
-│   └── wuhan.rev.2.bt2
-├── version
-└── wf
-    └── __init__.py
-```
+### Options
+
+#### `--template`, `-t`
+
+One of `r`, `conda`, `subprocess`, `empty`. If not provided, user will be prompted for input.
+
+#### `--dockerfile`, `-d`
+
+Generate a Dockerfile for the workflow.
 
-This example workflow is ready for registration (see below).
+#### `--cuda`
```

Comment:
these really need to be one parameter like `--base-image`

Reply:
also templates AND base images is a bit redundant, need a way to explain the difference and decide what cause where